models/fates/R/model2netcdf.FATES.R

```diff
@@ -10,581 +10,143 @@
 ##' @name model2netcdf.FATES
 ##' @title Code to convert FATES netcdf output into into CF standard
 ##'
-##' @param outdir Location of FATES model output
+##' @param outdir Location of FATES model output (e.g. a path to a single ensemble output)
+##' @param sitelat Latitude of the site
+##' @param sitelon Longitude of the site
+##' @param start_date Start time of the simulation
+##' @param end_date End time of the simulation
+##' @param pfts a named vector of PFT numbers where the names are PFT names
+##' @param settings pecan settings object
+##' @param process_partial should failed runs be processed? Defaults to `FALSE`.
+##'   `TRUE` will generate .nc files for runs that have generated some, but not
+##'   all, of the expected outputs
+##'
 ##' 
 ##' @examples  
 ##' \dontrun{
 ##' example.output <- system.file("case.clm2.h0.2004-01-01-00000.nc",package="PEcAn.FATES")
 ##' model2netcdf.FATES(outdir="~/")
 ##' }
+##'
+##' @author Michael Dietze, Shawn Serbin, Rob Kooper, Toni Viskari, Istem Fer
+## modified M. Dietze 07/08/12 modified S. Serbin 05/06/13
+## refactored by Istem Fer on 03/2018
```

Comment:
I suppose you copy-pasted this from some place else, I haven't touched this code before, please revise this to list the original authors (Michael Dietze, Shawn Serbin) and yourself